img/logo/logo-responsive.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M21.8529 3.63082C11.6733 3.63082 3.39493 11.9231 3.39493 22.1176C3.39493 32.3104 11.6733 40.6018 21.8529 40.6018C32.0308 40.6018 40.3117 32.3104 40.3117 22.1176C40.3117 11.9231 32.0308 3.63082 21.8529 3.63082ZM21.8529 44C9.80334 44 -0.000244141 34.1829 -0.000244141 22.1176C-0.000244141 10.0497 9.80334 0.232666 21.8529 0.232666C33.9016 0.232666 43.7069 10.0497 43.7069 22.1176C43.7069 34.1829 33.9016 44 21.8529 44ZM21.8538 10.5117C15.453 10.5117 10.2656 15.7087 10.2656 22.1175C10.2656 28.5246 15.453 33.7191 21.8538 33.7191C28.2529 33.7191 33.4411 28.5246 33.4411 22.1175C33.4411 15.7087 28.2529 10.5117 21.8538 10.5117ZM21.8538 5.63104L25.679 9.40827L30.9374 8.3032L32.1153 13.5536L37.1371 15.4684L35.295 20.5208L38.4849 24.8523L34.2058 28.1066L34.5514 33.4746L29.195 33.8977L26.5874 38.6014L21.8538 36.0554L17.1201 38.6014L14.5125 33.8977L9.15608 33.4746L9.50168 28.1066L5.22349 24.8523L8.41249 20.5208L6.56956 15.4684L11.5905 13.5536L12.7701 8.3032L18.0276 9.40827L21.8538 5.63104ZM16.9427 22.1169C16.9427 24.828 19.1456 27.0339 21.8538 27.0339C24.5611 27.0339 26.7632 24.828 26.7632 22.1169C26.7632 19.405 24.5611 17.1991 21.8538 17.1991C19.1456 17.1991 16.9427 19.405 16.9427 22.1169ZM14.6342 22.1169C14.6342 18.1307 17.8713 14.8857 21.8538 14.8857C25.8345 14.8857 29.0734 18.1307 29.0734 22.1169C29.0734 26.104 25.8345 29.3473 21.8538 29.3473C17.8713 29.3473 14.6342 26.104 14.6342 22.1169Z" fill="#00343D"/>
+</svg>
```

Comment:
These SVGs are not optimized, you can use these app
For SVG: https://imageoptim.com/mac
For PNG, JPEG and among others: https://tinypng.com/
@Jefer-Max